react/components/PickupDetailsModal.tsx

```diff
@@ -100,7 +100,7 @@ const PickupDetailsModal: React.VFC<Props> = ({
             <FormattedMessage id="store/checkout.shipping.pickupPointsModal.address" />
           </p>
 
-          <PlaceDetails />
+          <PlaceDetails customAddress={pickupOption.address} />
```

Comment:
can't we just wrap this component in a `AddressContext` component instead of creating this new prop?

Reply:
I don't think this new prop is a good API for this component, but I'd gladly talk with you if you want to (or I could just post here too haha) :-)

Reply:
so do you think its ok to wrap an AddressContext inside another AddressContext? Tbh it looks ok, but just making sure

Reply:
and btw, I like your solution - it's one less prop to handle :blob-party:

Reply:
> so do you think its ok to wrap an AddressContext inside another AddressContext?

yep! totally fine

Reply:
done!